src/selectedtests/test_mappings/create_test_mappings.py

```diff
@@ -59,19 +61,19 @@ def generate_test_mappings(
 
     source_re = re.compile(source_file_pattern)
     test_re = re.compile(test_file_pattern)
-    module_source_re = None
-    module_test_re = None
-    if module_name:
-        module_source_re = re.compile(module_source_file_pattern)
-        module_test_re = re.compile(module_test_file_pattern)
-
     most_recent_module_commit = None
     with TemporaryDirectory() as temp_dir:
         test_mappings_list, most_recent_project_commit = generate_project_test_mappings(
             evg_api, evergreen_project, temp_dir, source_re, test_re, project_commit_limit
         )
 
-        if module_name:
+        if module_name is not None:
+            if module_source_file_pattern is None or module_test_file_pattern is None:
+                raise ValueError(
+                    "You must specify module_source_file_pattern and module_test_file_pattern when passing a module"
+                )
+            module_source_re = re.compile(module_source_file_pattern)
+            module_test_re = re.compile(module_test_file_pattern)
```

Comment:
I think the logic is basically equivalent, and makes mypy happy.

Reply:
You can remove this line as well, right? `most_recent_module_commit = None`

Reply:
A test fails when I remove `most_recent_module_commit = None`.

Reply:
Talked IRL, nvm